02-GitHub.Rmd

```diff
@@ -9,67 +9,74 @@ opts_chunk$set(tidy.opts=list(width.cutoff=50), tidy = FALSE)
+If you're a fancy pants programmer you may want to use git from the command line (e.g., terminal or bash depending on your situation) so you can look cool, and admittedly have more fine-grained control over the process. But if you're like the rest of us, and if you're reading this document right now, you may want a more user- and beginner-friendly option.
 
 ```{r image1, echo=FALSE, out.width="700px", fig.align='center'}
 knitr::include_graphics("images/Git0.PNG")
 ```
 
 Image credit: devmountain.com
 
-## .git the language {#Git2.3}
+## The Language of Git {#Git2.3}
 
-Were you the kid that was tormented by your siblings speaking Pig Latin (also known as "Igpay Atinlay") and no matter how hard you tried, you couldn't figure it out. Learning to use .git and Github can feel like learning a new language. But we promise it is easier than Pig Latin.
+Were you the kid that was tormented by your siblings speaking Pig Latin (also known as "Igpay Atinlay") and no matter how hard you tried, you couldn't figure it out? Learning to use Git and Github can feel like learning a new language. But we promise it is easier than Pig Latin.
 
-When working with .git and GitHub, you are going to see the words `repo`, `origin`, `commit`, `push`, `pull`, `clone`, `fork`, and `branch`, among others. We will start with some basic definitions here, and in later chapters will elaborate on terms as necessary.
+When working with Git and GitHub, you are going to see the words `repo`, `origin`, `commit`, `push`, `pull`, `clone`, `fork`, and `branch`, among others. We will start with some basic definitions here, and in later chapters will elaborate on terms as necessary.
 
 **repo(sitory)** - a folder whose contents are is tracked by Git. A repo can exist locally (on your computer) as a remote repository (on GitHub), or both.
 
-**origin** - the primary .git repo, against which other copies are compared. In our examples, we will create the origin repo in GitHub
+**origin** - the primary Git repo, against which other copies are compared. In our examples, we will create the origin repo in GitHub
 
-**push** - sends changes from your local .git repo to the remote repository (e.g. GitHub)
+**push** - sends changes from your local Git repo to the remote repository (e.g. GitHub)
 
-**pull** - gets changes from the cloud and updates your local .git repo
+**pull** - gets changes from the cloud and updates your local Git repo
 
 > Note: this is different from a `pull request` which is when you want someone else to incorporate your changes into their GitHub repo. More on this later.
 
-**commit** - create a named version of a set of one or more changes to a local .git repo, to be pushed to the cloud. Also used to refer to this named state. In other words, when you commit your changes, you create a commit.
+**commit** - create a named version of a set of one or more changes to a local Git repo. 
```

Comment:
Technically commits can be completely local, and still be useful. So they don't *have* to be pushed.